weatherAPI/controllers/weather.go

```diff
@@ -0,0 +1,32 @@
+package controllers
+
+import (
+	"API/weatherAPI/libraries"
+
+	"github.com/astaxie/beego/logs"
+
+	"github.com/astaxie/beego"
+)
+
+type WeatherController struct {
+	beego.Controller
+}
+
+//@Title Get Json Response
+//@Description Get the Response from the endpoint of weather api
+//@Param city query string true "name of the City Example: Bogota"
+//@Param country query string true "Country is a country code of two characters in lowercase. Example: co"
+//@Success 200 {object} "Success"
+//@Failure 400 Bad Request
+//@Failure 400 Not Found
+//@Accept json
+//@router /r [get]
+func (wc *WeatherController) Get() {
+	city := wc.GetString("city")
+	country := wc.GetString("country")
```

Comment:
What happen if the parameters `city` or `country` are not sent.
We need to handle this kind of errors.

Reply:
Errro handled it